config.yml.default

```diff
@@ -157,32 +157,33 @@ greeting:
   #   {me} to insert bot's name
   # Any other words in curly braces will be removed.
   hello: "Hi! I'm {me}. Good luck! Type !help for a list of commands I can respond to." # Message to send to opponent chat at the start of a game
-  goodbye: "Good game!" # Message to send to opponent chat at the end of a game
-  hello_spectators: "Hi! I'm {me}. Type !help for a list of commands I can respond to." # Message to send to spectator chat at the start of a game
+  goodbye: "Good game, well played!" # Message to send to opponent chat at the end of a game
+  hello_spectators: "Hi! I'm {me} and I'm playing against {opponent}. Type !help for a list of commands I can respond to." # Message to send to spectator chat at the start of a game
   goodbye_spectators: "Thanks for watching!" # Message to send to spectator chat at the end of a game
 
 # pgn_directory: "game_records" # A directory where PGN-format records of the bot's games are kept
 
 matchmaking:
-  allow_matchmaking: false    # Set it to 'true' to challenge other bots.
-  challenge_variant: "random" # If set to 'random', the bot will choose one variant from the variants enabled in 'challenge.variants'.
-  challenge_timeout: 30       # Create a challenge after being idle for 'challenge_timeout' minutes. The minimum is 1 minute.
-  challenge_initial_time:     # Initial time in seconds of the challenge (to be chosen at random).
+  allow_matchmaking: true          # Set it to 'true' to challenge other bots.
+  challenge_variant: "random"      # If set to 'random', the bot will choose one variant from the variants enabled in 'challenge.variants'.
+  challenge_timeout: 30            # Create a challenge after being idle for 'challenge_timeout' minutes. The minimum is 1 minute.
+  challenge_initial_time:          # Initial time in seconds of the challenge (to be chosen at random).
     - 60
-    - 120
-  challenge_increment:        # Increment in seconds of the challenge (to be chosen at random).
+    - 180
+  challenge_increment:             # Increment in seconds of the challenge (to be chosen at random).
     - 1
     - 2
-#  challenge_days:            # Days for correspondence challenge (to be chosen at random).
+    - 0
```

Comment:
I don't think 0 should be added by default because it could lead to timeouts.

Reply:
Removed.